modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -15,9 +15,128 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Prerequisites
 
-* A ROSA OpenShift Cluster with the required access and tokens.
+* A ROSA OpenShift Cluster with the required access and tokens. See procedure in *Preparing AWS account* below.
 * link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
 
+.Preparing AWS account
+An AWS account must be ready to accept an OADP installation.
+
+. Create the following environment variables.
```

Comment:
can add info on where to create saying 'by running the following command..'
Guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
applied, thanks